src/applications/vaos/services/organization.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Functions in here should map a var-resources API request to a similar response from
+ * a FHIR resource request
+ */
+import { VHA_FHIR_ID } from '../utils/constants';
+import { getParentFacilities } from '../api';
+import { mapToFHIRErrors } from '../utils/fhir';
+
+/**
+ * Maps /vaos/facilities?facility_codes[]=983&facility_codes[]=984 to
+ * /Organization?identifier=983,984
+ *
+ * @param {Array} parentFacilities A list of parent facilities from var-resources
+ * @returns {Object} A FHIR searchset of Organization resources
+ */
+function mapParentFacilities(parentFacilities) {
```

Comment:
@narin Do you think we should put these in separate files?

Reply:
I think so.  The service files will end up getting pretty large so having them separate could be a little bit cleaner.